rasa/utils/log_utils.py

```diff
@@ -0,0 +1,93 @@
+import os
+import logging
+import sys
+from typing import Any, Dict
+
+import structlog
+from structlog_sentry import SentryProcessor
+from rasa.shared.exceptions import RasaException
+from rasa.plugin import plugin_manager
+
+
+FORCE_JSON_LOGGING = os.environ.get("FORCE_JSON_LOGGING")
+LOGGING_LEVEL = os.environ.get("LOGGING_LEVEL", "INFO")
```

Comment:
The logging level can actually be set via the cmdline also, see [here](https://github.com/RasaHQ/rasa/blob/97072e882c47dcd807ca2c74e0cff7d01686a3b5/rasa/__main__.py#L101), maybe this can be passed over to `configure_structlog` as a parameter. You can also reuse [this logic](https://github.com/RasaHQ/rasa/blob/97072e882c47dcd807ca2c74e0cff7d01686a3b5/rasa/utils/common.py#L187) in case log_level is None.